src/server/routes/edit.route.js

```diff
@@ -52,20 +52,24 @@ const editRouter = () => {
           );
           throw err;
         }
-        if (req.params.originator === "la-selector") {
+        if (
+          req.params.originator === "la-selector" &&
+          Object.keys(response.validatorErrors).length === 0
+        ) {
+          req.session.localAuthority = await getCouncilDataByID(
+            +req.body.local_authority
+          );
+          // If the local authority not onboarded and has a registration form URL, redirect to it instead of the normal path
```

Comment:
as above - comment to show why not on-boarded. As an aside, I hate this edit/new controller thing - we ned separate controllers per page